Description: fix warning "incompatible implicit declaration of built-in function 'free'"
 by unconditionally including stdlib.h (required on Linux)
Author: Florian Schlichting <fsfs@debian.org>

--- a/parser.c
+++ b/parser.c
@@ -1,8 +1,6 @@
 #include "parser.h"
 #include<stdio.h>
-#ifdef DARWIN
-  #include "stdlib.h"
-#endif
+#include "stdlib.h"
 #ifdef NOSTRING
   void memset(char *s, int c, int n) {
     char *se = s + n;
